x/ibc-transfer/keeper/relay.go

```diff
@@ -70,17 +71,41 @@ func (k Keeper) SendTransfer(
 
 	// begin createOutgoingPacket logic
 	// See spec for this logic: https://github.com/cosmos/ics/tree/master/spec/ics-020-fungible-token-transfer#packet-relay
-
 	channelCap, ok := k.scopedKeeper.GetCapability(ctx, host.ChannelCapabilityPath(sourcePort, sourceChannel))
 	if !ok {
 		return sdkerrors.Wrap(channeltypes.ErrChannelCapabilityNotFound, "module does not own channel capability")
 	}
 
+	// NOTE: denomination and hex hash correctness checked during msg.ValidateBasic
+	prefixedDenom := token.Denom
```

Comment:
I find `prefixedDenom` to be slightly confusing because technically "ibc/" is also a prefix. I don't have any good alternative suggestions though

Reply:
`hashPrefixedDenom` ?